lib/puppet/pops/model/ast.rb

```diff
@@ -4755,7 +4755,8 @@ def self.register_pcore_types
     Model::SelectorEntry,
     Model::SelectorExpression,
     Model::NamedAccessExpression,
-    Model::Program]
+    Model::Program
+  ]
```

Comment:
This file is autogenerated via the generate_ast_model rake task https://github.com/puppetlabs/puppet/blob/272abb97efa5d1c44fd238e3988b5a9f7e87cd3f/tasks/generate_ast_model.rake#L85 Maybe just ignore `lib/puppet/pops/model/ast.rb`?

Reply:
Thanks for the review. I've added a new commit to exclude this file from Rubocop checks and have rolled back the changes I'd made to the file as well.